14/umbraco-workflow/getting-started/workflow-content-app.md

```diff
@@ -0,0 +1,99 @@
+# Content App
+
+Umbraco Workflow adds a [Content App](https://docs.umbraco.com/umbraco-cms/extending/content-apps) to all content nodes in the **Content** section where a workflow is enabled. The Workflow content app includes three sub-sections:
```

Comment:
Yes, you are right.. But I guess, Nathan will update the article later once v14 is due for release. Currently, I only added the article back in cz it was missing from v14. Since I made some changes for v13 I wanted to make those in v14 as well so as to not loose those changes🙈

Reply:
Aaaah! Missed that part 😅 Thought it was a whole new article 🙈 

Let's leave that for now then, yes 😁